dir.common.props

```diff
@@ -95,4 +95,41 @@
     <CrossTargetComponentFolder Condition="'$(BuildArch)' == 'arm' and '$(TargetsLinux)' == 'true'">x64</CrossTargetComponentFolder>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
+    <UseSharedCompilation>true</UseSharedCompilation>
+  </PropertyGroup>
+
+  <!-- Set the kind of PDB to Portable -->
+  <PropertyGroup>
+    <DebugType Condition="'$(DebugType)' == ''">Portable</DebugType>
+  </PropertyGroup>
+
+  <!-- Common properties -->
+  <PropertyGroup>
+
+    <!-- This name is used to create a GIT repository URL https://github.com/dotnet/$(GitHubRepositoryName) used to find source code for debugging -->
+    <GitHubRepositoryName Condition="'$(GitHubRepositoryName)' == ''">coreclr</GitHubRepositoryName>
+
+    <Platform Condition="'$(Platform)' == ''">$(BuildArch)</Platform>
+    <Platform Condition="'$(Platform)' == 'amd64'">x64</Platform>
+
+    <!-- Default to portable build if not explicitly set -->
+    <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
+
+    <OverrideRestoreOutputPath>true</OverrideRestoreOutputPath>
+
+    <!-- If true, indicates that this is not an officially supported release -->
+    <!-- It is important to flip this to false in official release branches -->
+    <!-- Keep it in sync with PRERELEASE in clrdefinitions.cmake -->
+    <IsPrerelease>true</IsPrerelease>
```

Comment:
This is already set in src/Directory.Build.props. If it needs to be set at this higher level instead, we should remove it from that file (which should already include this file by way of the root Directory.Build.props).

Reply:
It isn't needed at a higher level, but it might (and was set at the root level by the way the pre-existing dir.props worked. If we ever have tests that depend on prerelease features, they would need this. I've simply removed it from src/Directory.Build.props which should be fine.